terra-jupyter-base/Dockerfile

```diff
@@ -88,10 +74,8 @@ ENV LC_ALL en_US.UTF-8
 # Python / Jupyter
 #######################
 
-ENV USER jupyter-user
-ENV UID 1000
-RUN useradd -m -s /bin/bash -N -u $UID $USER
-
+ENV USER jupyter
+RUN adduser jupyter sudo && usermod -g users jupyter
```

Comment:
Want to make sure I understand this command.. does this create a `juypter `and `sudo` user? Is `sudo` possible?

Reply:
if I remember correctly, the main thing is `jupyter` user's group needs to be `users`..can't remember if `sudo` is needed

Reply:
removed `sudo`, and it seems like it's working